src/StackAdmin/Resources/Commands.Resources/AzureRM.Resources.psd1

```diff
@@ -74,62 +75,70 @@ NestedModules = @('.\Microsoft.Azure.Commands.Resources.dll',
+               'New-AzureRmADGroup', 'Remove-AzureRmADGroup', 
+               'Get-AzureRmADServicePrincipal', 'Get-AzureRmADSpCredential', 
+               'Get-AzureRmADUser', 'New-AzureRmADAppCredential', 
+               'New-AzureRmADSpCredential', 'New-AzureRmADUser', 
+               'Remove-AzureRmADAppCredential', 'Remove-AzureRmADApplication', 
+               'New-AzureRmADApplication', 'Remove-AzureRmADServicePrincipal', 
+               'New-AzureRmADServicePrincipal', 'Remove-AzureRmADSpCredential', 
+               'Remove-AzureRmADUser', 'Update-AzureRmADApplication', 
+               'Update-AzureRmADServicePrincipal', 'Update-AzureRmADUser', 
+               'Remove-AzureRmResourceGroup', 'Get-AzureRmProviderFeature', 
+               'Register-AzureRmProviderFeature', 'Get-AzureRmLocation', 
+               'Export-AzureRmResourceGroup', 'Get-AzureRmResourceProvider', 
+               'Register-AzureRmResourceProvider', 
+               'Unregister-AzureRmResourceProvider', 
+               'Get-AzureRmResourceGroupDeployment', 
+               'New-AzureRmResourceGroupDeployment', 
+               'Remove-AzureRmResourceGroupDeployment', 
+               'Stop-AzureRmResourceGroupDeployment', 
+               'Test-AzureRmResourceGroupDeployment', 'Set-AzureRmResourceGroup', 
+               'New-AzureRmResourceGroup', 'Get-AzureRmResourceGroup', 
+               'Save-AzureRmResourceGroupDeploymentTemplate', 
+               'Get-AzureRmResourceGroupDeploymentOperation', 
+               'Get-AzureRmResourceLock', 'Invoke-AzureRmResourceAction', 
+               'Move-AzureRmResource', 'New-AzureRmResourceLock', 
+               'Get-AzureRmPolicyAssignment', 'Get-AzureRmPolicyDefinition', 
+               'Get-AzureRmPolicySetDefinition', 'New-AzureRmPolicyAssignment', 
+               'New-AzureRmPolicyDefinition', 'New-AzureRmPolicySetDefinition', 
+               'Remove-AzureRmPolicyAssignment', 'Remove-AzureRmPolicyDefinition', 
+               'Remove-AzureRmPolicySetDefinition', 'Set-AzureRmPolicyAssignment', 
+               'Set-AzureRmPolicyDefinition', 'Set-AzureRmPolicySetDefinition', 
+               'Remove-AzureRmResource', 'Remove-AzureRmResourceLock', 
+               'Set-AzureRmResource', 'New-AzureRmResource', 
+               'Set-AzureRmResourceLock', 'Get-AzureRmResource', 
+               'Get-AzureRmManagedApplicationDefinition', 
+               'New-AzureRmManagedApplicationDefinition', 
+               'Set-AzureRmManagedApplicationDefinition', 
+               'Remove-AzureRmManagedApplicationDefinition', 
+               'Get-AzureRmManagedApplication', 'New-AzureRmManagedApplication', 
+               'Set-AzureRmManagedApplication', 'Remove-AzureRmManagedApplication', 
+               'Get-AzureRmManagementGroup', 'New-AzureRmManagementGroup', 
+               'Update-AzureRmManagementGroup', 'Remove-AzureRmManagementGroup', 
```

Comment:
Remove the management group related cmdlets from the exported cmdlet list as they are not supported in AzureStack